cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -27,12 +27,14 @@
 <h1>{{ value.heading }}</h1>
 <div class="lead-paragraph">{{ parse_links(value.intro) | safe }}</div>
 {{ parse_links(value.body) | safe }}
-<ul class="list list__links">
-    {% for link in value.links %}
-        <li class="list_item">
-            <a class="list_link" href="{{ link.url }}">
-                {{ link.text }}
-            </a>
-        </li>
-    {% endfor %}
-</ul>
+{% if value.links.url %}
```

Comment:
Where's the bit in /careers/ where this is done? Trying to find it..
Do we have a way to test this when the links value has a URL? I worried it may always end up being false and never show, even when we want it to.